src/main/resources/META-INF/resources/index.ftl

```diff
@@ -45,11 +45,11 @@
 <#assign startTime = .now?time>
 <#assign isRoot = folder.root>
 
-<!-- was username. homeDir is populated if the home
-directory for that user actually exists -->
-<#if homeDir??>
-  <#assign homeURL = '${contextPath}list/${username}'>
-</#if>
+<#--<!-- was username. homeDir is populated if the home-->
+<#--directory for that user actually exists &ndash;&gt;-->
+<#--<#if homeDir??>-->
+<#--  <#assign homeURL = '${contextPath}list/${username}'>-->
+<#--</#if>-->
```

Comment:
This can just be deleted if it's in the `_main.ftl` and that works properly.

Reply:
Good catch - missed cleaning that up.